tests/kernel/profiling/profiling_api/testcase.yaml

```diff
@@ -3,3 +3,8 @@ tests:
     arch_exclude: nios2 riscv32
     platform_exclude: em_starterkit
     tags: kernel
+  kernel.profiling.initstack:
+    arch_exclude: nios2 riscv32
+    platform_exclude: em_starterkit
+    extra_configs:
+      - CONFIG_INIT_STACKS=n
```

Comment:
This option is off by default.
Did you intend to set this to "y"?
How are we testing that the stack was properly filled with 0xaa?
Not seeing this doing much that's useful.
And why are you excluding nios2/riscv32/em_starterkit? That makes no sense, this is supposed to work on all arches, and I know for sure that Nios II supports this.